### ISSUE TITLE
Implement the Graph API changes wireframe designs

### ISSUE THREAD

**Issue by adhiambovivian** [microsoftgraph/microsoft-graph-explorer-v4#1175]

**Comment by darrelmiller** [comment]
The original intent was to use "sparkle" decorators in the existing UX to indicate changes to Graph that are context sensitive.

**Comment by ddyett** [comment]
@adhiambovivian is this blocked on getting the wireframe or actionable work on engineering.

**Comment by adhiambovivian** [comment]
@ddyett  this needs PM direction ie if it's a feature we want to add to Graph Explorer

**Comment by ddyett** [comment]
ok moving to PM review.

**Comment by RabebOthmani** [comment]
@ddyett This was a feature request before I join I believe. Feel free to leave the ticket as a bookmark or remove it if it makes the board cleaner. This feature is not part of the current roadmap. I can re-create an issue if in the future we decide to implement it. Thanks!

**Comment by lehrt** [comment]
Hi all, this issue is almost 4 years stale. Graph Explorer is no longer in active development and has moved to maintenance mode. No more major features like this are funded. If you are an Microsoft employee, you can reach out to internal Graph Explorer team about new tasks/feature requests. Closing this issue.
